lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -172,44 +371,228 @@ static void deriveBodyEquatable_enum_eq(AbstractFunctionDecl *eqDecl) {
                                       AccessSemantics::Ordinary,
                                       fnType);
   }
-
+  
   TupleExpr *abTuple = TupleExpr::create(C, SourceLoc(), { aIndex, bIndex },
                                          { }, { }, SourceLoc(),
                                          /*HasTrailingClosure*/ false,
                                          /*Implicit*/ true);
-
+  
   auto *cmpExpr = new (C) BinaryExpr(cmpFuncExpr, abTuple, /*implicit*/ true);
   statements.push_back(new (C) ReturnStmt(SourceLoc(), cmpExpr));
-
+  
   BraceStmt *body = BraceStmt::create(C, SourceLoc(), statements, SourceLoc());
   eqDecl->setBody(body);
 }
```

Comment:
Please check your editor's whitespace setting and remove the above space changes.